File: C/medium/reverse list.c
#include <stdio.h>

int main() {
    int n, i;
    printf("Enter number of elements: ");
    scanf("%d", &n);
    int arr[n];
    printf("Enter %d numbers: ", n);
    for (i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }
    printf("Reversed list:");
    for (i = n - 1; i >= 0; i--) {
        printf(" %d", arr[i]);
    }
    printf("\n");
    return 0;
}

File: JAVA/medium/reverse list.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter numbers separated by space: ");
        String[] numbers = scanner.nextLine().split(" ");
        System.out.print("Reversed list:");
        for (int i = numbers.length - 1; i >= 0; i--) {
            System.out.print(" " + numbers[i]);
        }
        System.out.println();
    }
}
